src/m_files.cc

```diff
@@ -1025,13 +1035,30 @@ void Instance::M_WriteEurekaLump(Wad_file *wad) const
 	if (!loaded.portName.empty())
 		lump->Printf("port %s\n", loaded.portName.c_str());
 
+        //remember pwd
+        static char old_dir[FL_PATH_MAX];
```

Comment:
No static variables allowed, as I'm seeking to make the program as parallel-safe as possible.